modules/claims_api/app/sidekiq/claims_api/v2/poa_form_builder_job.rb

```diff
@@ -18,7 +18,7 @@ class PoaFormBuilderJob < ClaimsApi::ServiceBase
       # it queues a job to update the POA code in BGS, as well.
       #
       # @param power_of_attorney_id [String] Unique identifier of the submitted POA
-      def perform(power_of_attorney_id, form_number, rep_id, action) # rubocop:disable Metrics/MethodLength
+      def perform(power_of_attorney_id:, form_number:, action:, rep_id: nil) # rubocop:disable Metrics/MethodLength
```

Comment:
[Sidekiq best practices](https://github.com/sidekiq/sidekiq/wiki/Best-Practices) warn against using keyword arguments, as they may not be demarshalled correctly when sidekick moves the argument data around. Positional arguments are fine (like the previous arguments for this job), as is passing in simple JSON blobs (e.g. an "opts" hash argument).

The signature below should work & still allow for the rep_id to default to nil:
```def perform(power_of_attorney_id, form_number, action, rep_id = nil)```